### PR TITLE
solves the problem of syncing multiple times send data, once

### PR DESCRIPTION
**Pull request by Ebed-meleck**
solves the problem of syncing multiple times to resend the data, once

### CODE REVIEW

File: lib/screens/home.dart [hunk 1]
```diff
@@ -245,43 +305,43 @@ class _HomePageState extends State<HomePage> {
...
-        fetchInventory(),
-        // update sync info
-        _saveSyncInfo(_formattedLastUpdate, _countSynced, _maxToSync)
-      ]);
 
-      await fetchLots();
+    syncLots()
+        .then((_) => syncMovementEntries())
+        .then((_) => syncStockMovementExists())
+        .then((_) => syncStockMovementExists())
```
Comment (mbayopanda) [inline lib/screens/home.dart]: @Ebed-meleck why did you call this procedure `syncStockMovementExists()` twice?

**Review by mbayopanda — APPROVED** [review]
@Ebed-meleck Good job, now the sync works as expected 👍🏽

File: lib/screens/home.dart (hunk 1 above)
Comment (Ebed-meleck) [reply lib/screens/home.dart]: In one go, it recovers just a single output movement (consumption) if there was a loss it does not return. Hence why I called him twice. To resend the output

**Review by mbayopanda — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**